webroot/src/components/Licensee/LicenseeList/LicenseeList.ts

```diff
@@ -169,8 +183,10 @@ class LicenseeList extends Vue {
             }
         });
 
-        // Temp for limited server paging support
-        if (this.licenseStore.error?.message === 'end of list' && page > 1) {
+        // Support for limited server paging support
+        if (this.licenseStore.error?.message === 'end of list') {
```

Comment:
I know this was already here, but any reason to not encode this as an "error type" of sorts? Seems brittle to do an error message comparison.

Reply:
Made this into a custom error class.